functoria/app/functoria_command_line.ml

```diff
@@ -51,6 +51,17 @@ let build_dir =
   Term.(const (function None -> None | Some f -> Some (Fpath.v f))
         $ Arg.(value & opt (some string) None & doc))
 
+let timestamp =
+  let doc =
+    Arg.info
+      ~docs:configuration_section
+      ~docv:"DATA"
+      ~doc:"The timestamp to use in generated files. If not provided, use \
+            the current date instead."
+      ["timestamp"]
+  in
+  Arg.(value & opt (some string) None & doc)
```

Comment:
Do we have to protect against code injection here? Maybe we should only accept valid rfc3339 timestamps?

Reply:
as it looks atm, passing `--timestamp "*)"` will result in a `main.ml` which is not valid OCaml code

Reply:
But you could pass `--timestamp "*) Sys.command \"echo toto\" ;; (*"`